### ISSUE TITLE
[Pimcore 5.0.2 Stable] Unable set a PIMCORE_ENVIRONMENT which is different than default ones

### ISSUE THREAD

**Issue by Yivan** [pimcore/pimcore#2265]
Downlading the last stable on the web site and running installation bring an error:

![error1b](https://user-images.githubusercontent.com/9025839/33187041-1e147ce4-d08f-11e7-83bd-465e6527a6a2.png)

Environnement is ok (i was able to install previous Pimcore 5 versions). Check requirement is ok (some warning, only Composer is missing).

Here the detail (if i refresh the installation page i got this page):

![error2](https://user-images.githubusercontent.com/9025839/33177547-3e212cde-d063-11e7-8e7e-cfdc4e01838f.png)

Just specificity of my environment:
Our ENV name is `development` (and not `dev`) so before install I copy config_dev.yml as config_development.yml. I don't know if it could be related.

**Comment by Yivan** [comment]
I tested by renaming our dev environement to `dev` and install working ok.

So this bug appears only when PIMCORE_ENVIRONMENT is different than default ones.
I changed the GitHub issue title.

**Comment by Yivan** [comment]
In fact the same problem occurs if you create a new environment in existing installation of Pimcore 5.0.2 stable.
I will update the title of the issue.

I managed to make it work a little further (bypassing the monolog error), by creating a file named "development.yml" (by copying dev.yml) here too:
dev-phileass.fr\pimcore\lib\Pimcore\Bundle\CoreBundle\Resources\config\pimcore
It seems this directory contains environment file and should be mentioned here for people setting new environments:
https://pimcore.com/docs/5.0.x/Development_Documentation/Deployment/Multi_Environment.html

I get now another error : (

```
 (2/2) FileLoaderLoadException
There is no extension able to load the configuration for "web_profiler" (in .../pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/config/pimcore/config_dev.yml). Looked for namespace "web_profiler", found "framework", "security", "twig", "monolog", "swiftmailer", "doctrine", "sensio_framework_extra", "cmf_routing", "pimcore", "pimcore_admin", "app" in .../pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/config/pimcore/config_dev.yml (which is being imported from ".../pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/config/pimcore/config_development.yml").
in FileLoader.php (line 177)
```

```
(1/2) InvalidArgumentException

There is no extension able to load the configuration for "web_profiler" (in .../pimcore/lib/Pimcore/Bundle/CoreBundle/Resources/config/pimcore/config_dev.yml). Looked for namespace "web_profiler", found "framework", "security", "twig", "monolog", "swiftmailer", "doctrine", "sensio_framework_extra", "cmf_routing", "pimcore", "pimcore_admin", "app"
```

Is there some other place i should declare/create file for `development` env ? (if yes should be documented i think) Or is it a bug ?

**Comment by Yivan** [comment]
Ok I got it!
I needed to register the bundles in AppKernel.

I will make a PR to give some clues about all of this in the documentation https://pimcore.com/docs/5.0.x/Development_Documentation/Deployment/Multi_Environment.html
around `If you add a new environment...`. It will help next developpers like me struggling with creating a new env!
Even if it is symfony standard, it is valuable information i think so pimcore developper have all information to set a new environment in one place in the documentation.

**This issue can be closed.**
